.changeset/weak-planes-rush.md

```diff
@@ -0,0 +1,5 @@
+---
+"@orbit-ui/components": patch
+---
+
+fix for tileLink borderRadius
```

Comment:
Cette ligne là va être des release notes. Prend le temps de mettre une meilleure ligne svp du genre: 

`fix: Illustrations in a TileLink now have proper border radius`